support/trace/tracer.go

```diff
@@ -0,0 +1,79 @@
+package trace
+
+// CarrierFormat defines the format of the propogation that needs to injected or extracted
+type CarrierFormat byte
+
+const (
+	// Binary represents trace context as opaque binary data.
+	Binary CarrierFormat = iota
+	// TextMap represents trace context as key:value string pairs.
+	//
+	// Unlike HTTPHeaders, the TextMap format does not restrict the key or
+	// value character sets in any way.
+	TextMap
+	// HTTPHeaders represents trace context as HTTP header string pairs.
+	//
+	// Unlike TextMap, the HTTPHeaders format requires that the keys and values
+	// be valid as HTTP headers as-is (i.e., character casing may be unstable
+	// and special characters are disallowed in keys, values should be
+	// URL-escaped, etc).
+	HTTPHeaders
+)
+
+// FlogoTracer interface to configure individual tracers
+type FlogoTracer interface {
+	// Name() returns the name of the registered tracer
+	Name() string
+
+	// configure() sets up the tracer by gathering required configuration from the trace config
+	// environment variable. The tracer must also register as an event listenerbefore tracing can
+	// begin.
+	configure() error
```

Comment:
Shouldn't this be Configure()

Reply:
I initially had the implementation under the same package and didn't want to expose the function at the activity level. Forgot to change it back once we decided to have different implementations. 
Changed now.